ATSAMD21/luos_hal.c

```diff
@@ -563,9 +566,7 @@ static void LuosHAL_GPIOInit(void)
     }
 
     /*Configure GPIO pin : TxPin */
-    PORT_REGS->GROUP[COM_TX_PORT].PORT_PINCFG[COM_TX_PIN] = PORT_PINCFG_RESETVALUE; // no pin mux / no input /  no pull / low streght
-    PORT_REGS->GROUP[COM_TX_PORT].PORT_PINCFG[COM_TX_PIN] |= PORT_PINCFG_PULLEN_Msk;
-    PORT_REGS->GROUP[COM_TX_PORT].PORT_PINCFG[COM_TX_PIN] |= PORT_PINCFG_PMUXEN_Msk;                   // mux en
+    PORT_REGS->GROUP[COM_TX_PORT].PORT_PINCFG[COM_TX_PIN] = PORT_PINCFG_RESETVALUE;                    // no pin mux / no input /  no pull / low streght
```

Comment:
you need to set up the disable config au init with : 
        PORT->Group[COM_TX_PORT].PINCFG[COM_TX_PIN].reg = PORT_PINCFG_RESETVALUE; // no pin mux / no input /  no pull / low streght
        PORT->Group[COM_TX_PORT].PINCFG[COM_TX_PIN].reg |= PORT_PINCFG_PULLEN;    // Enable Pull
        PORT->Group[COM_TX_PORT].OUTSET.reg = (1 << COM_TX_PIN);                  // Pull up